### PR TITLE
Requirement of the invoicing sample rate

### PR DESCRIPTION
**Pull request by corvus-ch**
Document the minimal sample rate required for invoicing.

## Checklist

- [x] Try to isolate changes into separate PRs (to build a better changelog).
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `change`, `decision`, `requirement/quality`, `requirement/functional`, `dependency`
      as they show up in the changelog

### CODE REVIEW

File: docs/modules/ROOT/pages/references/quality-requirements/functional/sampelrate-for-invoicing.adoc [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Rationale::
+The scrape interval of Prometheus is only an approximate number.
+Depending on the system load, scrapes can be delayed.
+This is important because the smallest unit of interest for invoicing is a Minute.
+With a scrape interval of 60s, a scrape might happen 63s later than the previous one.
+If that happens, we miss a sample.
+Ideally a scrape interval smaller than 60s is used.
+This should guarantee the one sample per minute.
+Downsampling of two or more samples is done using the statistical mean.
```
Comment (ccremer) [inline docs/modules/ROOT/pages/references/quality-requirements/functional/sampelrate-for-invoicing.adoc:29]: I had an idea to reword it a bit to separate the implementation ideas.
```suggestion
Response::
A time series of usage metrics

Response measure::
At least one sample per minute

Rationale::
The smallest resolution an invoicing adapter can expect is one sample per minute.
Ideally a scrape interval smaller than 60s is used.
This should guarantee the one sample per minute.
Downsampling of two or more samples is done using the statistical mean.

[TIP]
====
The scrape interval of Prometheus is only an approximate number.
Depending on the system load, scrapes can be delayed.
With a scrape interval of 60s, a scrape might happen 63s later than the previous one.
If that happens, we miss a sample.
====
```

File: docs/modules/ROOT/partials/nav-reference.adoc [hunk 2]
```diff
@@ -12,6 +12,9 @@
 
 * Quality Requirements
 
+** Functional
+*** xref:appuio-cloud:ROOT:references/quality-requirements/functional/sampelrate-for-invoicing.adoc[Sample rate for invoicing]
```
Comment (ccremer) [inline docs/modules/ROOT/partials/nav-reference.adoc:16]:
```suggestion
** Functional
*** xref:appuio-cloud:ROOT:references/quality-requirements/functional/samplinginterval-for-invoicing.adoc[Sampling interval for invoicing]
```

File: docs/modules/ROOT/pages/references/quality-requirements/functional/sampelrate-for-invoicing.adoc [hunk 3]
```diff
@@ -0,0 +1,29 @@
+= Minimal sample rate for invoicing
```
Comment (ccremer) [inline docs/modules/ROOT/pages/references/quality-requirements/functional/sampelrate-for-invoicing.adoc:1]: Probably same title as nav menu item?
```suggestion
= Sampling interval for invoicing
```

**Review by ccremer — APPROVED** [review]

File: docs/modules/ROOT/partials/nav-reference.adoc [hunk 4]
```diff
@@ -13,7 +13,7 @@
 * Quality Requirements
 
 ** Functional
-*** xref:appuio-cloud:ROOT:references/quality-requirements/functional/sampelrate-for-invoicing.adoc[Sample rate for invoicing]
+*** xref:appuio-cloud:ROOT:references/quality-requirements/functional/samplinginterval-for-invoicing.adoc[Sampling interval for invoicing]
```
Comment (ccremer) [inline docs/modules/ROOT/partials/nav-reference.adoc]: You also need to rename the file :)

File: docs/antora.yml [hunk 5]
```diff
@@ -6,6 +6,7 @@ nav:
   - modules/ROOT/partials/nav.adoc
 asciidoc:
   attributes:
+    global: APPUiO Global
```
Comment (tobru) [inline docs/antora.yml:9]:
```suggestion
    global: APPUiO Cloud Global
```

APPUiO is a bit too unspecific, it could also be APPUiO Managed

**Pull request merged**